src/ralph/conf.py

```diff
@@ -181,22 +200,19 @@ class ParserSettings(BaseModel):
 class Settings(BaseSettings):
     """Represents Ralph's global environment & configuration settings."""
 
-    class Config:
+    class Config(BaseSettingsConfig):
         """Pydantic Configuration."""
 
-        case_sensitive = True
-        env_nested_delimiter = "__"
-        env_prefix = "RALPH_"
-        env_file = APP_DIR / ".env"
-        env_file_encoding = getattr(io, "LOCALE_ENCODING", "utf8")
+        env_file = core_settings.APP_DIR / ".env"
```

Comment:
The default path to the dot env file is expected to be `.env` by convention, _i.e._ a dot file in the current directory. Dot env files are supposed to be set on a per-project basis, not globally for the application.

Reply:
Thanks) I wasn't aware of the convention; good to know) Updated this part in this [fixup commit](https://github.com/openfun/ralph/commit/aab3122288f5c5c25962fa62c0a165b979ade7df))